labs/lab5/wordpress-template.yaml

```diff
@@ -57,22 +57,8 @@ objects:
               - MYSQL_PWD="$DBPASS" mysql -h 127.0.0.1 -u $DBUSER -D $DBNAME
                 -e 'SELECT 1'
             timeoutSeconds: 2
-          volumeMounts:
-          - mountPath: /run/mariadb
-            name: mariadb-volume-1
-          - mountPath: /var/lib/mysql
-            name: mariadb-volume-2
-          - mountPath: /var/log/mariadb
-            name: mariadb-volume-3
```

Comment:
we don't need any of these any longer?

Reply:
this was previously done for perm issues when deploying to openshift. when using "emptydir" mounts openshift would automatically set the appropriate perms for these critical dirs w/ a restricted deployment and arbitrary uid. we've since moved to non-root images and we're setting perms appropriately at build time for a smooth openshift deployment -
https://github.com/dustymabe/summit-2018-container-lab/blob/3b2a7ae6c2cb180ca9333582fa63df6ef1f961ec/labs/lab3/mariadb/Dockerfile.reference#L12-L14

these volumes were just tmpfs mounts inside the container anyway. what would be great in the future is a persistent volume for mariadb at /var/lib/mysql.